Gopkg.toml

```diff
@@ -101,7 +101,7 @@ ignored = [
   name = "honnef.co/go/tools"
   # This revision is not yet on master (as of 05-12-2019); when it is, we can
```

Comment:
Update comment. We should just set it to master instead of a fixed sha.

Reply:
I used `branch = "master"` instead but then various things break because there have been API changes. I updated the comment but I'll leave it pinned at the revision since I just wanted to fix vendoring, not bump this. Let me know if I should've done something different and I'll follow up.